API.md

```diff
@@ -169,6 +169,24 @@ Draw
 
 ---
 
+### `.set(Object: featureCollection) -> [featureId, featureId]`
+
+This function takes a featureCollection and performs the required delete, create and update actions internally to make Draw represent this change. Effectively this is the same as `Draw.deleteAll()` followed by `Draw.add(featureCollection)` except that it doesn't effect performance as much.
+
+Example:
+
+```js
+var ids = Draw.set({type: 'FeatureCollection', features: [{
+  type: 'Feature',
+  properties: {},
+  id: 'example-id',
+  geometry: { type: 'Point', coordinates: [0, 0] }
+}]});
+// => ['example-id']
+```
+
```

Comment:
> Does it only accept FeatureCollections?

It only accepts FeatureCollections

> How is it intended to deal with partial features, e.g. could I do

If the feature doesn't have an id it will create a new feature. If a feature doesn't have properties its invalid and an error will be thrown.